docker-compose.yml

```diff
@@ -245,6 +245,39 @@ services:
     volumes:
       - "redisinsight:/db"
 
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.8.1
+    environment:
+      - node.name=modularhistory-es
```

Comment:
@alashow, where do these settings come from?  Can you add a comment with a link to documentation, if available?